src/components/tooltip/Tooltip.scss

```diff
@@ -84,15 +84,15 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
             border-top-color: inherit;
         }
 
-        &.is-error::after {
+        &.bdl-is-error::after {
             top: 100%;
             left: 50%;
             margin-left: -$arrow-size + 1;
             border-top-color: $bdl-watermelon-red-10;
         }
     }
 
-    .tooltip-target-attached-top.tooltip-target-attached-left > & {
```

Comment:
Does Tether respect the new root class name when applying these generated classes?

Reply:
Yes